modules/xmpp/RtxModifier.js

```diff
@@ -120,17 +120,30 @@ export default class RtxModifier {
 
             return sdpStr;
         }
+
+        return this.modifyRtxSsrcs2(videoMLine)
+            ? sdpTransformer.toRawSDP() : sdpStr;
+    }
+
+    /**
+     * Does the same thing as {@link modifyRtxSsrcs}, but takes the
+     *  {@link MLineWrap} instance wrapping video media as an argument.
+     * @param {MLineWrap} videoMLine
+     * @return {boolean} <tt>true</tt> if the SDP wrapped by
+     *  {@link SdpTransformWrap} has been modified or <tt>false</tt> otherwise.
+     */
+    modifyRtxSsrcs2(videoMLine) {
```

Comment:
well, but it does the same except that takes different arguments ? so I see that as ver2 (because JS does not distinguish between argument types)